### PR TITLE
Updated rakuten.co.jp

### LINKED ISSUE #5824: Update rakuten.co.jp

### PR DESCRIPTION
**Pull request by luixxiul**
- Added information of nonfunctional domains, problematic domains, etc.
- Added domains linked from www.rakuten.co.jp/sitemap/
- Added domains on www.infoseek.co.jp

Closes #5824

### CODE REVIEW

**Comment by luixxiul** [comment]
This patch is a big one, but please check it as soon as possible since rakuten.co.jp is 8th popular site in Japan. Thanks.

**Comment by luixxiul** [comment]
WIP, adding other domains by checking on chrome based on each test url.

**Comment by luixxiul** [comment]
Ready for review

**Comment by luixxiul** [comment]
Please make sure the latest commit eac54cd depends on 35a1df0.

**Comment by luixxiul** [comment]
Is there any problem which prevents this from being merged? If so please let me know.

File: src/chrome/content/rules/rakuten-static.com.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		- jp.rakuten-static.com *
+
+	* akamai
+
+-->
+
+<ruleset name="rakuten-static.com (partial)">
+
+	<target host="image.card.jp.rakuten-static.com" />
+	<target host="plaza.jp.rakuten-static.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/rakuten-static.com.xml]: can you provide valid test urls here?

File: src/chrome/content/rules/rakuten-static.com.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!--
+
+	For other Rakuten coverage, see Rakuten.co.jp.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/rakuten-static.com.xml]: You should therefore list all related rules in the Rakuten.co.jp.xml file.

File: src/chrome/content/rules/rakuten-static.com.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+	Problematic domains:
+
+		- a.ichiba.jp.rakuten-static.com *
+		- jp.rakuten-static.com *
+
+	* akamai
```
Comment (J0WI) [inline src/chrome/content/rules/rakuten-static.com.xml]: Akamai should work.

**Comment by J0WI** [comment]
There are various mixed active content issues on deep links, like page of a single product. See e.g. on books or video.

**Comment by luixxiul** [comment]
Thanks for reviewing. I will split the ruleset into ones with and without mixed contents.

File: src/chrome/content/rules/rakuten-static.com.xml (hunk 2 above)
Comment (luixxiul) [inline src/chrome/content/rules/rakuten-static.com.xml]: Done [here](https://github.com/EFForg/https-everywhere/pull/5827/files#diff-26ed1b519bef0e9172c56c2156ca0641R10).

**Comment by J0WI** [comment]
Thanks for this huge contribution!
I tested some random sites and didn't find any further issues, so lgtm.

**Comment by luixxiul** [comment]
And this should be the result: https://twitter.com/luixxiul01/status/767763617173245953

**Comment by luixxiul** [comment]
Also the other rulesets listed [here](http://github.com/EFForg/https-everywhere/pull/5827/files#diff-26ed1b519bef0e9172c56c2156ca0641R7) will be coming soon.

**Comment by luixxiul** [comment]
It turned out that there are bunch of Rakuten Groups websites. You can see the list of site in the footer of http://global.rakuten.com/corp/worldwide/

**Pull request merged**